src/server/rot.rs

```diff
@@ -0,0 +1,161 @@
+// Copyright lowRISC contributors.
+// Copyright (c) 2021 Western Digital
```

Comment:
Manticore shouldn't be under WD's copyright. Sounds like a CLA issue? @asb @domrizz0

Reply:
Just to clarify, per the standard lowRISC (and OpenTitan) [CLA](https://github.com/lowRISC/manticore/blob/master/CLA), contributors retain copyright. This is really about whether we maintain metadata about who contributed to a file in its header - our standard approach (used here, and in github.com/lowrisc/opentitan) is not to do so, because invariably it gets out of date and there are questions about whether a change is large enough to merit a new copyright line etc etc.

Alistair - any questions just let me or Dom know, and we can vector to the legal committee as appropriate.

Reply:
(Just so I don't miss this in the future I'm setting up CI w the standard lowrisc checker, at least to flag these issues so we can sort them explicitly.)

Reply:
I'm a little confused. It sounds like it should be Copyright WD as contributors retain copyright, is that correct?

Reply:
"Copyright lowRISC contributors" aims to catch that the work is copyrighted by contributors to this lowRISC-hosted project (intentionally slightly non-specific because moving code between repos isn't uncommon). This is not dissimilar to e.g. Go and Fuschia who use "The Go Authors" and "The Fuchsia Authors" respectively.